packages/jbrowse-web/src/plugins/AddTrackDrawerWidget/components/ConfirmTrack.js

```diff
@@ -163,7 +163,14 @@ export function guessAdapter(fileName, protocol) {
     return {
       type: unsupported,
     }
-  return {}
+
+  if (/\/trackData.json$/i.test(fileName))
+    return {
+      type: 'NCListAdapter',
+      rootUrlTemplate: fileName
+    }
+
+    return {}
 }
 
 function guessTrackType(adapterType) {
```

Comment:
Is this going to be a BasicTrack? If so, probably want to add a `NCListAdapter: 'BasicTrack'` line to `guessTrackType`.

Reply:
👍 done